team-manager-api/src/main/java/ua/knu/knudev/teammanagerapi/request/DepartmentCreationRequest.java

```diff
@@ -0,0 +1,20 @@
+package ua.knu.knudev.teammanagerapi.request;
+
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.*;
+import lombok.Builder;
+import ua.knu.knudev.teammanagerapi.dto.SpecialtyCreationDto;
+
+import java.util.Set;
+
+@Builder
+public record DepartmentCreationRequest(
+        @NotEmpty(message = "Department name cannot be blank or empty.")
+        @Size(min = 3, max = 150, message = "Too long or short department name.")
+        @Pattern(regexp = "^[A-Za-zА-Яа-яЇїІіЄєҐґ\s-]+$", message = "Department name contains invalid characters.")
```

Comment:
make and test pattern only for ukrainan language. And put message that department name must contain only urkainain characters

Reply:
done